.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
```

Comment:
What formatting rules is this using? Is it similar to the one we agreed on: https://github.com/eclipse-ee4j/cargotracker#contributing?

Reply:
Try to use a pre commit hook to format Java codes, but I do not know why the build pipeline still reject it.

Reply:
I also tried to use IDEA Google Format plugin to reformat the codes

Reply:
Can you please look into it? Also, don't we need only one and not both? I recently had to update the GitHub Actions dependencies. Perhaps that's causing an issue?